### PR TITLE
Use react router history.push

### PR DESCRIPTION
**Pull request by luisrudge**
Following our sample PR: https://github.com/auth0-samples/auth0-react-samples/pull/154

### CODE REVIEW

File: articles/quickstart/spa/react/_includes/_centralized_login.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!-- markdownlint-disable MD002 MD041 MD034 -->
 
-<%= include('../../_includes/_login_preamble', { library: 'React' }) %>
+<%= include('../../\_includes/\_login_preamble', { library: 'React' }) %>
```
Comment (lbalmaceda) [inline articles/quickstart/spa/react/_includes/_centralized_login.md]: rollback this line change

File: articles/quickstart/spa/react/_includes/_centralized_login.md [hunk 2]
```diff
@@ -128,7 +141,7 @@ This is a set of custom [React hooks](https://reactjs.org/docs/hooks-intro.html)
 
 The next few sections will integrate these hooks into the various components that make up the app.
 
-<%= include('../../_includes/_silent-auth-social-idp') %>
+<%= include('../../\_includes/\_silent-auth-social-idp') %>
```
Comment (lbalmaceda) [inline articles/quickstart/spa/react/_includes/_centralized_login.md]: rollback this line change

**Review by lbalmaceda — CHANGES_REQUESTED** [review]

File: articles/quickstart/spa/react/_includes/_centralized_login.md (hunk 1 above)
Comment (luisrudge) [reply articles/quickstart/spa/react/_includes/_centralized_login.md]: done

File: articles/quickstart/spa/react/_includes/_centralized_login.md (hunk 2 above)
Comment (luisrudge) [reply articles/quickstart/spa/react/_includes/_centralized_login.md]: done

**Comment by joshcanhelp** [comment]
@luisrudge - Looks like this one fell off the radar. I requested review from @lbalmaceda again but need to rebase.

**Review by lbalmaceda — DISMISSED** [review]
Fix the conflicts @luisrudge

**Comment by luisrudge** [comment]
done

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**